annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/out/MethodRecord.java

```diff
@@ -53,15 +55,20 @@ public class MethodRecord {
   private URI uri;
   /** Unique id assigned to this method across all visited methods. */
   private final int id;
+  /**
+   * Set of annotations on the method return type. If the method has no annotations, then this field
```

Comment:
I think technically these are annotations either on the method return type or on the method itself (in the case of a declaration annotation)?  Probably best to fix the comment?